JuliaLowering/test/scopes.jl

```diff
@@ -58,23 +58,281 @@ begin
 end
 """) === :outer_y
 
-# wrap expression in scope block of `scope_type`
-function wrapscope(ex, scope_type)
-    g = JuliaLowering.ensure_attributes(ex._graph, scope_type=Symbol)
-    ex = JuliaLowering.reparent(g, ex)
-    makenode(g, ex, K"scope_block", [ex._id], [:scope_type=>scope_type])
+#=
+| old\new: || global     | local | arg                | sparam             |
+|----------++------------+-------+--------------------+--------------------|
+| global   || no-op      | (*)   |                    |                    |
+| local    || error (*)  | no-op |                    |                    |
+| arg      || shadow(??) | error | error (not unique) |                    |
+| sparam   || shadow(??) | error | error (sparam/arg) | error (not unique) |
+=#
+@testset "Conflicts in the same local scope" begin
+
+    # no-op cases.  It would probably be clearer (but breaking) if these were
+    # errors like the conflict cases (two of the same decl should never do
+    # anything, and the user might be expecting two variables).
+    @testset "global,global" begin
+        s = "function (); global g; global g; 1; end"
```

Comment:
JuliaLowering seems to accept this example, but flisp does not:
```julia
julia> JuliaLowering.include_string(TestMod,"""
       function (); global g; const g = 1; end
       """)
#anon###0 (generic function with 1 method)

julia> function (); global g; const g = 1; end
ERROR: syntax: `global const` declaration not allowed inside function around REPL[5]:1
Stacktrace:
 [1] top-level scope
   @ REPL[5]:1
```

somehow `global g` gets JL to look the other way